soh/soh/Enhancements/randomizer/LockOverworldDoors.cpp

```diff
@@ -0,0 +1,156 @@
+    {{ SCENE_KAKARIKO_VILLAGE, 7615 },           RAND_INF_KAK_POTION_SHOP_UNLOCKED }, // Child Day/Night and Adult Day
+    {{ SCENE_KAKARIKO_VILLAGE, 2495 },           RAND_INF_BOSS_HOUSE_UNLOCKED },
+    {{ SCENE_KAKARIKO_CENTER_GUEST_HOUSE, 447 }, RAND_INF_BOSS_HOUSE_UNLOCKED },
+    {{ SCENE_KAKARIKO_VILLAGE, 3750 },           RAND_INF_GRANNYS_POTION_SHOP_UNLOCKED }, // Child
+    {{ SCENE_KAKARIKO_VILLAGE, 3519 },           RAND_INF_GRANNYS_POTION_SHOP_UNLOCKED }, // Adult
+    {{ SCENE_POTION_SHOP_GRANNY, 447 },          RAND_INF_GRANNYS_POTION_SHOP_UNLOCKED },
+    {{ SCENE_KAKARIKO_VILLAGE, 5567 },           RAND_INF_SKULLTULA_HOUSE_UNLOCKED },
+    {{ SCENE_HOUSE_OF_SKULLTULA, 447 },          RAND_INF_SKULLTULA_HOUSE_UNLOCKED },
+    {{ SCENE_KAKARIKO_VILLAGE, 1471 },           RAND_INF_IMPAS_HOUSE_UNLOCKED },
+    {{ SCENE_IMPAS_HOUSE, 447 },                 RAND_INF_IMPAS_HOUSE_UNLOCKED },
+    {{ SCENE_KAKARIKO_VILLAGE, 447 },            RAND_INF_WINDMILL_UNLOCKED },
+    {{ SCENE_WINDMILL_AND_DAMPES_GRAVE, 2495 },  RAND_INF_WINDMILL_UNLOCKED },
+    {{ SCENE_KAKARIKO_VILLAGE, 4543 },           RAND_INF_KAK_SHOOTING_GALLERY_UNLOCKED }, // Day
+    {{ SCENE_KAKARIKO_VILLAGE, 4751 },           RAND_INF_KAK_SHOOTING_GALLERY_UNLOCKED }, // Night
+    {{ SCENE_SHOOTING_GALLERY, 447 },            RAND_INF_KAK_SHOOTING_GALLERY_UNLOCKED },
+    {{ SCENE_GRAVEYARD, 645 },                   RAND_INF_DAMPES_HUT_UNLOCKED }, // Child Day
+    {{ SCENE_GRAVEYARD, 447 },                   RAND_INF_DAMPES_HUT_UNLOCKED }, // Child Evening & Adult
+    {{ SCENE_GRAVEYARD, 774 },                   RAND_INF_DAMPES_HUT_UNLOCKED }, // Child Night (After Dampes Tour)
+    {{ SCENE_GRAVEKEEPERS_HUT, 447 },            RAND_INF_DAMPES_HUT_UNLOCKED },
+    {{ SCENE_LON_LON_RANCH, 2495 },              RAND_INF_TALONS_HOUSE_UNLOCKED },
+    {{ SCENE_LON_LON_RANCH, 2473 },              RAND_INF_TALONS_HOUSE_UNLOCKED },
+    {{ SCENE_LON_LON_RANCH, 2729 },              RAND_INF_TALONS_HOUSE_UNLOCKED },
+    {{ SCENE_LON_LON_BUILDINGS, 1471 },          RAND_INF_TALONS_HOUSE_UNLOCKED },
+    {{ SCENE_LON_LON_RANCH, 1471 },              RAND_INF_STABLES_UNLOCKED },
+    {{ SCENE_STABLE, 447 },                      RAND_INF_STABLES_UNLOCKED },
+    {{ SCENE_LON_LON_RANCH, 447 },               RAND_INF_BACK_TOWER_UNLOCKED },
+    {{ SCENE_LON_LON_BUILDINGS, 447 },           RAND_INF_BACK_TOWER_UNLOCKED },
+    {{ SCENE_LAKE_HYLIA, 447 },                  RAND_INF_HYLIA_LAB_UNLOCKED },
+    {{ SCENE_LAKESIDE_LABORATORY, 447 },         RAND_INF_HYLIA_LAB_UNLOCKED },
+    {{ SCENE_LAKE_HYLIA, 1471 },                 RAND_INF_FISHING_HOLE_UNLOCKED },
+    {{ SCENE_FISHING_POND, 447 },                RAND_INF_FISHING_HOLE_UNLOCKED },
+};
+
+static void OnDoorInit(void* actorRef) {
+    EnDoor* enDoor = static_cast<EnDoor*>(actorRef);
+    enDoor->randomizerInf = RAND_INF_MAX;
+
+    SPDLOG_INFO("TEST");
+
```

Comment:
Assume we don't need this one anymore ;)

Reply:
```suggestion
```